auth0/resource_auth0_log_stream_test.go

```diff
@@ -0,0 +1,192 @@
+package auth0
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("auth0_log_stream", &resource.Sweeper{
+		Name: "auth0_log_stream",
+		F: func(_ string) error {
+			api, err := Auth0()
+			if err != nil {
+				return err
+			}
+			l, err := api.LogStream.List()
+			if err != nil {
+				return err
+			}
+			for _, logstream := range l {
+				if strings.Contains(logstream.GetName(), "Test") {
+					log.Printf("[DEBUG] Deleting logstream %v\n", logstream.GetName())
+					if e := api.LogStream.Delete(logstream.GetID()); e != nil {
+						multierror.Append(err, e)
+					}
+				}
+			}
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	})
+}
+
+func TestAccLogStreamHttp(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: `
+				resource "auth0_log_stream" "my_log_stream" {
+					name = "Acceptance-Test-LogStream-http"
+					type = "http"
+					sink {
+						http_endpoint = "https://example.com/webhook/logs"
+						http_content_type = "application/json"
+						http_content_format = "JSONLINES"
+						http_authorization = "AKIAXXXXXXXXXXXXXXXX"
+					}
+				}
+				`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "name", "Acceptance-Test-LogStream-http"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "type", "http"),
+					//resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "status", "active"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_endpoint", "https://example.com/webhook/logs"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_type", "application/json"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_content_format", "JSONLINES"),
+					resource.TestCheckResourceAttr("auth0_log_stream.my_log_stream", "sink.0.http_authorization", "AKIAXXXXXXXXXXXXXXXX"),
+				),
+			},
+		},
+	})
+}
+func TestAccLogStreamEventBridge(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: `
+				resource "auth0_log_stream" "my_log_stream" {
+					name = "Acceptance-Test-LogStream-aws"
+					type = "eventbridge"
+					sink {
+						aws_account_id = "999999999999"
+						aws_region = "us-west-2"
+					}
+				}
```

Comment:
We would preferably follow the convention here and define them as variables. Should also add randomization also so we can be able to run tests in parallel.